.circleci/config.yml

```diff
@@ -237,6 +238,11 @@ jobs:
 
               while [ ! -f ~/miniconda/pytorch_installed ]; do sleep 2; done # wait for Pytorch
               pytest
+
+              #re-install with bullet and run physics tests again
+              rm -r build
```

Comment:
I don't believe this is needed

Reply:
Yeah -- if you don't `rm` it, the dependencies wouldn't need to be rebuilt :)

Reply:
Fortunately, ccache to the rescue, but still unneeded.